### PR TITLE
Incorporate Kelly's edits into Multiple Domains article

### LINKED ISSUE #1434: Full audit: How to serve multiple domains using virtual hosts
Full audit needed.

https://github.com/rackerlabs/rackspace-how-to/blob/3a1798e3176cb5f98dd53154ec38a9de95d74f06/content/cloud-servers/how-to-serve-multiple-domains-using-virtual-hosts.md

This article ranks 25 in most viewed articles.

### PR DESCRIPTION
**Pull request by DonNateR**
- Fixes #1434

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7bcb82dbd2/how-to/)

File: content/cloud-servers/how-to-serve-multiple-domains-using-virtual-hosts.md [hunk 1]
```diff
@@ -1,66 +1,28 @@
 ---
-permalink: how-to-serve-multiple-domains-using-virtual-hosts/
-audit_date:
+permalink: serving-multiple-domains-by-using-virtual-hosts
```
Comment (stephamon) [inline content/cloud-servers/how-to-serve-multiple-domains-using-virtual-hosts.md]: permalink needs forward slash

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d9dea329a6/how-to/).

File: content/cloud-servers/serve-multiple-domains-using-virtual-hosts.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+		</VirtualHost>
+
+		<VirtualHost \*:80>
+		  ServerName vh2.example.com
+		  DocumentRoot /var/www/vhosts/vh2
+		</VirtualHost>
+
+#### Element missing from VirtualHost directive
+
+If running httpd -S" reports the following error message:
```
Comment (stephamon) [inline content/cloud-servers/serve-multiple-domains-using-virtual-hosts.md]: httpd -S" should be `httpd -S`

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-923d2932b3/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**